app/views/submissions/_submissions_table.js.jsx.erb

```diff
@@ -307,31 +326,13 @@
         }.bind(this)
       });
     },
-    collectAllSubmissions: function() {
-      jQuery.ajax({
-        method: 'GET',
-        url: 'collect_all_submissions',
-        success: function(data) {
-          if(data.message){
-            this.props.onSuccess(data.message);
-          }
-          if(data.error){
-            this.props.onError(data.error);
-          }
-        }.bind(this),
-        error: function(xhr, status, text) {
-          var error = text + ': ' + xhr.responseText;
-          this.props.onError(error);
-        }.bind(this)
-      });
-    },
-    collectSectionSubmissions: function() {
+    collectSubmissions: function() {
       var dataLoad = {
-        sections: this.refs.sectionSelect.state.value.split(',')
+        groupings: this.props.selected_submissions
       };
       jQuery.ajax({
         method: 'GET',
```

Comment:
I didn't think of this before, but please make this a `POST` request. The data will sometimes be too big to send in a header.